.github/workflows/benchmark_commits.yml

```diff
@@ -122,16 +124,22 @@ jobs:
 
       - name: Benchmark given commit
         if: github.event_name != 'pull_request_target' || inputs.run_all_benchmarks == true
-        shell: bash -el {0}
+        shell: bash -l {0}
         run: | 
           git config --global --add safe.directory .
-          python -m asv run -v --show-stderr --bench $SUITE ${{ inputs.commit }}^!
-        
+          python -m asv run --show-stderr --durations all --bench $SUITE ${{ inputs.commit }}^!
```

Comment:
My custom timing script seems to underestimate how long it takes to run some benchmarks (although the ordering looks correct, which is still useful). I've since discovered ASV's built in `--durations all` which will be good to compare against.

I've removed the `-v` as otherwise the logs are just too hard to comprehend.